aip/general/0009.md

```diff
@@ -102,6 +102,14 @@ The version of an API or a group of APIs if they are defined
 together. An API Version is often represented by a string, such as "v1", and
 presents in API requests and Protocol Buffers package names.
 
+### Client
+
+A consumer of the API. Examples of clients include:
```

Comment:
"Consumer" is kind of vague.

How about "code that calls the API"?

There are also at-rest use cases, though, so maybe add "or operates on the resource data at rest".

Reply:
good call! done. Added an at-rest example.